nomad/structs/structs.go

```diff
@@ -4105,6 +4106,7 @@ type JobListStub struct {
 	ID                string
 	ParentID          string
 	Name              string
+	Namespace         string `json:",omitempty"`
```

Comment:
https://github.com/hashicorp/nomad/blob/master/api/jobs.go#L866
`api.JobListStub` should be updated as well, for `api` users.